mockup/patterns/moment/pattern.js

```diff
@@ -86,7 +86,42 @@ define([
   'mockup-i18n',
   'moment'
 ], function($, Base, i18n, moment) {
-  lazyLoadMomentLocale();
+  var currentLanguage = (new i18n()).currentLanguage;
+
+  // From https://github.com/moment/moment/blob/3147fbc/src/test/moment/format.js#L463-L468
+  var MOMENT_LOCALES =
+    'ar-sa ar-tn ar az be bg bn bo br bs ca cs cv cy da de-at de dv el ' +
+    'en-au en-ca en-gb en-ie en-nz eo es et eu fa fi fo fr-ca fr-ch fr fy ' +
+    'gd gl he hi hr hu hy-am id is it ja jv ka kk km ko lb lo lt lv me mk ml ' +
+    'mr ms-my ms my nb ne nl nn pl pt-br pt ro ru se si sk sl sq sr-cyrl ' +
+    'sr sv sw ta te th tl-ph tlh tr tzl tzm-latn tzm uk uz vi zh-cn zh-tw';
+
+  lazyLoadMomentLocale(currentLanguage);
```

Comment:
I would move this invocation after the function is defined and would call it without an argument (currentLanguage can be evaluated in the scope of lazyLoadMomentLocale, see comment above).
Not a merge blocker.

Reply:
You're right. I'll change that.

About the function declaration since it's hoisted should be fine but I'll double check, probably your suggestion makes things more readable.

Reply:
I pushed the 2 suggested changes but I must confess I wasn't too sure about the 2nd one. It seems to me not relying on local scope would be safer, for instance the function could become easier to test and also move around, for example, if we wanted it to go in a `moment-locale` pattern (i don't like bloating the pattern file, in this case it was OK-ish because it's little code but I'd prefer if it looked more similar to the others: define wrapper + extend Base + return extended object. yet I preferred to tackle that in a next PR)